### ISSUE TITLE
update to angular-l10n problems

### ISSUE THREAD

**Issue by damienbod** [robisim74/angular-l10n#54]
Hi Roberto

I'm trying to update to the latest version, everything builds now, but nothing is rendered.

Any ideas here?

https://github.com/damienbod/Angular2LocalizationAspNetCore/tree/updatingLocalization

Greetings Damien

**Comment by robisim74** [comment]
Hi Damien,

I'm seeing that you are using APP_INITIALIZER, but you forgot the following lines in the constructor of the app component:

```
this.translation.init();
this.locale.setDefaultLocale('de', 'CHF');
```
Try to remove them. If it still doesn't work, later I'll try your branch.

Let me know.

**Comment by damienbod** [comment]
still the same :(, thanks

**Comment by robisim74** [comment]
Hi Damien,

I made a pr to the branch.
The error was due to the fact that in a service accidentally you injected _Localization_ class instead of _LocaleService_ class (the first is not injectable, and this generated the incomprehensible Angular error).
I then updated the event subscription _languageCodeChanged_ to _defaultLocaleChanged_, because you are using the _defaultLocale_, and not only the language.

Greetings.

Roberto
